### PR TITLE
README: Update to latest behavior: we no longer need addfunds command.

### PR DESCRIPTION
**Pull request by ZmnSCPxj**
Fixes: #565

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,19 +67,29 @@ cli/lightning-cli newaddr
 
 # Returns a transaction id <txid>
 bitcoin-cli -testnet sendtoaddress <address> <amount>
+```
 
-# Retrieves the raw transaction <rawtx>
-bitcoin-cli -testnet getrawtransaction <txid>
+Eventually `lightningd` will be able to receive the funds, it just
+requires `lightningd` catching up to the tip and confirming it on a
+block.
```
Comment (rustyrussell) [inline README.md]: Maybe just write:

    `lightningd` will register the funds once the transaction is confirmed.

**Comment by ZmnSCPxj** [comment]
Updated as requested.

**Comment by cdecker** [comment]
ACK 94cf5b2

> nit: wrap lines to 80 columns

I personally always go back and forth between wrapping at 80 chars or newline per sentence. I you have long paragraphs a single character change can cascade through the entire paragraph when wrapping at a fixed length. I found it much easier to diff when it's a line per sentence.

**Comment by ZmnSCPxj** [comment]
A compromise I found that mostly works is to wrap at both 80 columns *and* end of sentence. While a single char change potentially infects multiple lines, it will at least not infect entire paragraphs.

**Comment by jb55** [comment]
As of what commit is this true? I still had to addfunds while testing on mainnet.

**Comment by jb55** [comment]
My first mainnet transaction wasn't picked up, but my second one was. There might be a bug lurking around.

**Comment by ZmnSCPxj** [comment]
Since txfilter got implemented, I think? @cdecker

**Comment by cdecker** [comment]
We should be catching all addresses that we generate, and correctly initialize the txfilter upon restart. @jb55 could you try to compare the txfilter with the output of the transaction?

**Pull request merged**